stacks-common/src/address/c32.rs

```diff
@@ -251,6 +249,12 @@ fn c32_decode_ascii(input_str: &str) -> Result<Vec<u8>, Error> {
         return Err(Error::InvalidCrockford32);
     }
 
+    // ASCII size is 62.5%  that of c32-encoded
```

Comment:
Similarly here, this could say:

```rust
    // c32-encoding encodes 5 bits into each character, while ASCII encodes
    // 8-bits into each character. So, the ASCII-encoded size should be
    // (c32 size) * 5 / 8.
```